salt/netapi/rest_cherrypy/app.py

```diff
@@ -2326,7 +2326,6 @@ def signal_handler(signal, frame):
                         logger.error(
                                 "Error: Salt event has non UTF-8 data:\n{0}"
                                 .format(data))
-                time.sleep(0.1)
```

Comment:
This is likely to suppress high CPU load on idle. I'd leave `sleep` here, but set to `0.01` instead.